wp-graphql.php

```diff
@@ -74,8 +74,17 @@ function graphql_init_appsero_telemetry() {
 
 	$client = new Appsero\Client( 'cd0d1172-95a0-4460-a36a-2c303807c9ef', 'WP GraphQL', __FILE__ );
 
-	// @phpstan-ignore-next-line
-	$client->insights()->add_plugin_data()->init();
+	// If the Appsero client is version 1.2.0 or higher, add plugin data
+	if (version_compare( $client->version, '1.2.1', '>=' ) ) {
+		// @phpstan-ignore-next-line
+		$client->insights()->add_plugin_data()->init();
+
+	// Else, initialize without the plugin data
+	} else {
```

Comment:
I'm not familiar with the Appsero API, but wondering if we can drop the else statement for a _tiny_ codecomplexity improvement e.g.

```php
if( conditional_check() ) { // see comment on version_compare()
  $client->insights()->add_plugin_data();
}

$client->insights()->init();
```

Reply:
@justlevine ya, we can do that. I'll flip it